ls1intum/Artemis
# Enhancements/user statistics

### Checklist

### Motivation and Context


Test data should not be considered in the charts, because this is not valid data. Also, a view-type between month and year displaying in weekly spans would be nice, because instructors usually plan in weeks.

### Description

Adds a new Tab named Quarter (Quartal) and removes the data from test courses.

### Test Coverage
StatisticsResource.java: 100%
StatisticsService.java: 91%

### Steps for Testing


1. Log in to Artemis
2. Navigate to Server Administration -> User statistics
3. Focus on the Quartal tab
4. Make sure no initial error are thrown
5. Keep in mind the values for the important bars
6. Create/participate/assess exams/exercises and make sure the corresponding bars in the charts change in the wanted way
7. Declare your course a test course and make sure the data is the same as in step 3.

### Screenshots


<img width="1404" alt="Screenshot 2020-12-29 at 16 58 30" src="https://user-images.githubusercontent.com/44401560/103296812-329f9100-49f7-11eb-8dae-cbcc39d0d2ec.png">
